topology/Cardiff University/Cardiff Computing Cluster/CardiffPRPCachingInfrastructure.yaml

```
@@ -24,3 +24,24 @@ Resources:
         Description: Cardiff PRP Caching Service
     AllowedVOs:
       - LIGO
+      
+  CARDIFF_UK_OSDF_CACHE:
+    Active: true
+    Description: Cardiff OSDF cache
+    ID: 1413
```

```suggestion
    ID: 1415
```
1413 is in use by another resource.